src/Util/Logger/FriendicaProcessor.php

```diff
@@ -17,22 +17,22 @@ class FriendicaProcessor implements ProcessorInterface
 
 	private $skipStackFramesCount;
 
+	private $skipClassesPartials;
+
 	private $skipFunctions = [
 		'call_user_func',
 		'call_user_func_array',
 	];
 
-	private $skipFiles = [
-		'Logger.php'
-	];
-
 	/**
 	 * @param string|int $level The minimum logging level at which this Processor will be triggered
+	 * @param array $skipClassesPartials An array of classes to skip during logging
 	 * @param int $skipStackFramesCount If the logger should use information from other hierarchy levels of the call
 	 */
-	public function __construct($level = Logger::DEBUG, $skipStackFramesCount = 0)
+	public function __construct($level = Logger::DEBUG, array $skipClassesPartials = array(), $skipStackFramesCount = 0)
 	{
 		$this->level = Logger::toMonologLevel($level);
+		$this->skipClassesPartials = array_merge(array('Monolog\\'), $skipClassesPartials);
```

Comment:
I took this from https://github.com/Seldaek/monolog/blob/ebb804e432e8fe0fe96828f30d89c45581d36d07/src/Monolog/Processor/IntrospectionProcessor.php#L46
It's more explicit and now the functionality of this Processor is almost the same as `IntrospectionProcessor` with the only difference that we don't log the class and just the basename